interpreter/core/core.py

```diff
@@ -1,8 +1,21 @@
 """
 This file defines the Interpreter class.
-It's the main file. `import interpreter` will import an instance of this class.
+running ```import interpreter``` followed by ```interpreter.create_interpreter(**kwargs)``` will create an instance of this class.
 """
-from interpreter.utils import display_markdown_message
+
+import json
+import appdirs
+import os
+from datetime import datetime
+from typing import (Optional,
+                     Union, 
+                     Iterator,
+                     Any,
+                     Callable,
+                     List,
+                     Dict
+                     )
+
```

Comment:
I'm all for adding types, but I don't think they are necessary for this PR, and end up creating more noise in the diffs. I think this kind of change is better handled in a separate PR.

Reply:
alright yeah. I do agree, I simply added them for testing; we can remove them.